doc/source/reference/routines.linalg.rst

```diff
@@ -5,6 +5,17 @@
 Linear algebra (:mod:`numpy.linalg`)
 ************************************
 
+NumPy wraps a linear algebra backend with convenience functions. The backend
```

Comment:
Perhaps  something like:  

"The NumPy linear algebra functions rely on BLAS and LAPACK to provide efficient low level implementations of standard linear algebra algorithms. Those libraries may be provided by NumPy itself using C versions of a subset of their reference implementations, but when possible highly optimized libraries that take advantage of specialized processor functionality are preferred. Examples of such libraries are MKL (TM), OpenBLAS, and ATLAS. Because those libraries are multithreaded and processor dependent, environmental variables and external packages such as threadpoolctl_ may be needed to control the number of threads or specify the processor architecture."

Reply:
adopted, thanks